CONTRIBUTORS

```diff
@@ -0,0 +1,17 @@
+List of meta-riscv contributors, sorted by Alphabetical order
+
+ * Alistair Francis <alistair.francis@wdc.com>
+ * Chin Huat Ang <chinhuat@gmail.com>
+ * Jeremy Puhlman <jpuhlman@mvista.com>
+ * Khem Raj <raj.khem@gmail.com>
+ * Mirza Krak <mirza.krak@endian.se>
+ * Moritz Fischer <moritz.fischer@ettus.com>
+ * Palmer Dabbelt <palmer@sifive.com>
+ * pino-kim <sungwon.pino@gmail.com>
```

Comment:
First, thanks to add my e-mail to CONTRIBUTE file. pino-kim and sungwon.pino is same peopele. would you please remove line number 10.?

Reply:
Done.

Reply:
@alistair23  Thanks.